### PR TITLE
Theme docs

### PR DESCRIPTION
**Pull request by prateekbh**
**!Merge after Migrate21.**
Adds theme page to documentation

Fixes num 11 on #263

### CODE REVIEW

File: docs/src/routes/theme/index.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+// Class
+export default class ThemePage extends Component {
+  render() {
+    return (
+      <div className={style["page-theme"]}>
+        <div className="mdc-typography--display1">Original documentation</div>
+        <div className="mdc-typography--body">
+          preact-material-components brings{" "}
+          <span className="strong">mdc-theme</span>as is, you can refer to its
```
Comment (filoozom) [inline docs/src/routes/theme/index.js]: I'll admit I haven't tested the PR, but isn't there a space missing after ```<span className="strong">mdc-theme</span>```? Maybe the style has a margin though, so not sure.

File: docs/src/routes/theme/index.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+          documentation
+          <a href="https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme">
+            {" "}
```
Comment (filoozom) [inline docs/src/routes/theme/index.js]: Guess this should be after ```documentation``` two lines up.

File: docs/src/routes/theme/index.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+// Material Components
+import Button from "preact-material-components/Button";
+
+// Components
+import ComponentTable from "../../components/component-table";
+import CodeBlock from "../../components/code-block";
+
+// Styles
+import style from "./style";
```
Comment (filoozom) [inline docs/src/routes/theme/index.js]: This file doesn't exist:
```
./src/routes/theme/index.js
Module not found: Error: Can't resolve './style' in 'preact-material-components\docs\src\routes\theme'
 @ ./src/routes/theme/index.js 20:0-28
 @ ./src/components/app/index.js
 @ ./src/index.js
 @ multi ./src/index.js
Build failed!
```

File: docs/src/routes/theme/index.js [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+            {" "}
+            here
+          </a>.
+        </div>
+
+        <div className="mdc-typography--display1">Changing colors</div>
+        <div className="mdc-typography--body">
+          With preact-material-components you can either choose to use the
+          bundled <span className="strong">.css </span>
+          files or the <span className="strong">.scss </span> files. Both have
```
Comment (filoozom) [inline docs/src/routes/theme/index.js]: Should remove space before ```</span>```, but it's just a tiny detail.

File: docs/src/routes/theme/index.js (hunk 2 above)
Comment (prateekbh) [reply docs/src/routes/theme/index.js]: didnt get this

File: docs/src/routes/theme/index.js (hunk 1 above)
Comment (prateekbh) [reply docs/src/routes/theme/index.js]: fixed

File: docs/src/routes/theme/index.js (hunk 3 above)
Comment (prateekbh) [reply docs/src/routes/theme/index.js]: fixed

File: docs/src/routes/theme/index.js (hunk 4 above)
Comment (prateekbh) [reply docs/src/routes/theme/index.js]: fixed

**Comment by prateekbh** [comment]
@filoozom fixed the gaps

**Pull request merged**